rtc/getting-started.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: Getting started with RTC
+title_nav: Getting started
+description: Getting started with RTC
+keywords: rtc
+---
+
+{% assign beta_feature = "The RTC plugin" %}
+{% assign pre-release_type = "Closed Beta" %}
+{% include misc/beta-note.md %}
+{{site.requires_5_6v}}<br>
+{{site.premiumplugin}}<br>
+
+## Prerequisites
+
+* A {{site.cloudname}} API key from [{{site.accountpage}}]({{site.accountsignup}}).
+* A JWT key generated at [{{site.accountpageurl}}]({{site.accountpageurl}}). Temporarily store the generated keys for use in the following procedure.
+
+## Basic RTC setup
+
+The following example shows how to get the RTC plugin up and running using hard coded values. For a production setup, the document ID and secret would be retrieved from your server. This examples also assumes that a JWT provider endpoint exists at '/jwt'. For information on setting up a JWT endpoint, see: [JWT authentication]({{site.baseurl}}/rtc/jwt-authentication/).
+
+```js
+const yourDocumentId = 'some-document-id';
+const yourSecretKey = 'your secret 🔑';
+
+tinymce.init({
+  selector: 'textarea',
+  plugin: 'rtc',
+  rtc_document_details_provider: () => Promise.resolve({ documentId: yourDocumentId }).
```

Comment:
@tylerkelly13 just wanted to double-check, but I'm guessing we're fine using ES6 examples here, given RTC isn't supported on IE 11?

Reply:
Worth noting the highlighter doesn't seem to like using this syntax very much (or at least isn't highlighting the destructured object variables).

Reply:
Correct. Although I prefer older syntax for readability purposes, this plugin won't work with IE11 and this helps to make that clear to people who skip the introduction section on supported browsers.

Re: syntax highlighting, yeah, seems it "gives up" when you have an object in a function in an object :man_shrugging: